extensions/amp-next-page/0.2/service.js

```diff
@@ -109,6 +110,20 @@ export class NextPageService {
    * @param {!AmpElement} element
    */
   build(element) {
+    // Get the separator and more box (and remove the provided elements in the process)
+    const separator = this.getSeparatorElement_(element);
+    const moreBox = this.getMoreBoxElement_(element);
+
+    // Prevent multiple amp-next-page on the same document
```

Comment:
It's written this way so that if there's another `amp-next-page` element, we hide its separator and recommendation box, however it'll no longer be necessary once #26296 lands (since it simply removes the entire `amp-next-page` element when it encounters it on subsequent pages). I'll revert it back to what it used to be.